### ISSUE TITLE
Problems with deployment on W2008R2

### ISSUE THREAD

**Issue by andy-soft** [parle-io/parle#50]
Hi there
I found this jewel, and decided to test it, and on my PC it worked fine using the local server tied internally to Visual Studio 2013, then I set it up on a local W2008R2 server, in a webApp directory:
/chat
Initially it did not work, but after renaming the start of the 3 sentences, that includes the whole .js code in the original html, it started to show the little chat-popup on the right bottom.
But, all the admin stuff did not work, I tried it once, on my computer, 
As I did the deployment to the server, using VS 2013 automated deployment, the encrypted passwords file was copied, but the pages did not accept any pwd, and hence I did put the correct passwords there!
The I deleted the password file, (and reading the code) it should re-create it, 
But it still kept asking for the password, and not the re-registration procedure.
May be I am not too familiarized with MVC4 ¿could this have been kept in the server's cache?
I tried to debug the code, and set breakpoints, and the code never reached the .CS code section on login ¿am I doing something wrong?
I believe I am a skilled programmer, used to VS2010 until now, 
I was heavily using .aspx web-forms, internal code (algorithms, AI, etc.) used old-fashion and complicated remoting and appdomain schema, but did only some minor ALAX code using JQuery never (until now) met SignalIR as a multi-protocol code-proxysm nor AngularJS and the other quirks used here.
I was using a W2003R2 server, adn had to upgrade, because it does not support ASP.NET 4.5, (only until 4.0) WTF!
¿any clue?
.

**Comment by dstpierre** [comment]
Maybe the app does not have permission to write to the file? If everything else is working and not the admin part, that's the reason I could see.

**Comment by andy-soft** [comment]
The app does have permission, but it never goes trough the proxy's part on the server, I did setup breakpoints under VS and for a reason I could not establish it worked on the development, but not on the server, I checked and the app can write on the file, also there is no way to reissue the password, based on the actual app demo. May be a cache or just a problem with the subdirectory addressing part of the .js and/or some part of the app. I could not check it.
Yesterday I saw a JS component called  "function hasStorage()" which always returns null on the VS and may be return something else on the server test. I don't know how to intercept this on debugging with VS, may be on the browser debugger.. I have to check!
Remember the server test is located inside a directory which contains the WebApp (the deployment was done with VS2013)
